packages/component-library/src/StackedAreaChart/StackedAreaChart.js

```diff
@@ -23,7 +23,7 @@ import {
   getDefaultDataSeriesLabels,
   getDefaultAreaStyle
 } from "../utils/chartHelpers";
-import CivicVictoryTheme from "../VictoryTheme/VictoryThemeIndex";
+import { VictoryTheme } from "../_Themes/index";
```

Comment:
Should set all of these components to have this as a default theme, and add a theme prop

Reply:
Great point! Will do

Reply:
Done 🎨